vllm/worker/model_runner.py

```diff
@@ -1510,9 +1510,12 @@ def capture_model(self, kv_caches: List[List[torch.Tensor]]) -> None:
                         graph_runner)
 
         end_time = time.perf_counter()
+        end_free_gpu_memory = torch.cuda.mem_get_info()[0]
         elapsed_time = end_time - start_time
+        cuda_graph_size = end_free_gpu_memory - start_free_gpu_memory
         # This usually takes < 10 seconds.
         logger.info("Graph capturing finished in %.0f secs.", elapsed_time)
+        logger.info("CUDA graph takes %.2f GiB", cuda_graph_size / (1024**3))
```

Comment:
use `vllm.utils.GiB_bytes`

Reply:
thanks! Will revise in the next commmit.